main.tf

```diff
@@ -64,14 +74,16 @@ module "aws_deploy-integration-eu-west-2" {
 }
 
 module "aws_deploy-next-eu-west-2" {
-  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v1.0.0"
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.1.0"
   env               = "next"
   bootstrap_version = "${var.bootstrap_version}"
   vault_role        = "ae-node"
   vault_addr        = "${var.vault_addr}"
 
-  static_nodes  = 1
-  spot_nodes    = 2
+  static_nodes   = 1
+  spot_nodes_min = 2
+  spot_nodes_max = 2
```

Comment:
`snapshot_filename = "mnesia_uat_v-1_latest.tgz"?

Reply:
next environment is not connected to the testnet

Reply:
ah, i didn't see it is next